### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by Sofinaye**
I use javascript objects and array to store the data
and I use dynamic html
I add the popup window functionality 
and also the remove popup functionality

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi @Sofinaye , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has non descriptive PR summary and lacks commits from your partner

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -1,20 +1,345 @@
...
+  {
+    titleTop: "Professional art",
+    titleBottom: "printing data",
+    description:
+      "A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard",
+    image: "./img/art1.png",
+    technologies: ["css", "html", "bootstrap"],
+    seeLive: "#",
+    seeSource: "#"
+  },
```
Comment (FrederickMih) [inline index.js]: - [ ] Let's kindly make sure the design of our popup strictly follows that of Figma(e.g., font, colors, images, text, margins). Let's check out the two screenshots below and make the necessary adjustments. See live, see source, technologies buttons are not showing up. Let the popup be populated with all the data from the project array. Please kindly fix the issues.

| Yours | Figma |
| --- | --- |
| yourpopup | Capture |

- [ ] Also, kindly check and compare your mobile design to that of Figma.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi 🙋 @Sofinaye  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!






## Highlights </img>


- PR is professional ✔️ 
- Linters checks passed ✔️


## Required Changes ♻️

- [ ] As stated in the requirements let's kindly make sure the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) linter is equally set up in this project repo.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -538,6 +537,104 @@ body {
...
+}
+
+.popup-disc {
+  font-family: "Inter", serif;
+  font-size: 16px;
+  color: #344563;
+  margin: 2.1rem 0;
+}
+
+.popup-disc-btn {
```
Comment (huseyinbiyik) [inline style.css:618]: As the previous reviewer requested, stick to the Figma template, decrease the size of popup and align buttons like below.
Screenshot 2023-02-11 at 11 02 08 PM
Screenshot 2023-02-11 at 11 00 17 PM

File: style.css [hunk 3]
```diff
@@ -788,6 +885,33 @@ body {
   .menu-section-active {
     width: 0;
   }
+
+  .popupBody {
```
Comment (huseyinbiyik) [inline style.css:893]: Kindly stick to the Figma template as the previous reviewer requested and fix the overflowing issue and add some space above buttons.
Screen Recording 2023-02-11 at 11 06 54 PM

File: index.html [hunk 4]
```diff
@@ -79,118 +80,8 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
...
-    <div class="all-project">
-      <div class="projects-primary">
-        <div class="projects-heading1">
-          <h3>Professional art <span>printing Data</span></p>
-        </div>
-        <div class="projects-disc1">
-          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
-        </div>
-        <ul class="project1-bttns">
-          <li><button type="button" class="project1-btns">html</button></li>
```
Comment (huseyinbiyik) [inline index.html:116]: As the previous reviewer requested please stick to the design provided and make sure all popups are working properly.

Screen Recording 2023-02-11 at 11 03 58 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 5]
```diff
@@ -1,20 +1,342 @@
...
+
+const addPopup = () => {
+  if (selectedProject) {
+    const popupContainer = document.createElement('div');
+    popupContainer.setAttribute('id', 'popup-container');
+    document.body.style.overflowY = 'hidden';
+    document.querySelector('.about-overly').style.display = 'none';
+    document.body.appendChild(popupContainer);
+    const popup = document.createElement('div');
+    popup.setAttribute('class', 'popup');
```
Comment (bobb-Rob) [inline index.js]: - [ ] As mentioned by the previous reviewer, the popup is not properly displaying. Currently it is not scrollable to enable users to see the text and button hidden down.

image

I suggest you add the overflow- [See tutorial](https://www.w3schools.com/css/css_overflow.asp) on how to do this.

The desktop view hides the popup heading
image


Please fix this

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @Sofinaye , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Sofinaye 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Great work and effort on making the requested changes from the previous reviewers and on the design overall 💯 

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**